### ISSUE TITLE
Q: Long shellScriptTitledButton to be a ticker line?

### ISSUE THREAD

**Issue by l0bz1k** [Toxblh/MTMR#377]
Is it possible a long process name (example: "Google Chrome Helper (Renderer)") that does not fit in the width, to be a ticker line? 
Now i get it cut off at the end. 
Thank you!

```
 {
     "type": "shellScriptTitledButton",
     "width": 140,
     "refreshInterval": 3,
     "source": {
         "inline": "ps Axrc|grep -A 1 PID|tail -1|awk -F' ' '{print$5,$6,$7,$8,$9,$10}'"
     },
     "align": "left",
     "background": "#000000",
     "bordered": false
 },
```

**Comment by HigH-HawK** [comment]
You have set a fix width, so it won't extend automatically, you could however exchange the fix width with `autoResize`, so it automatically resizes to the length of the process name

```
{
   "type": "shellScriptTitledButton",
   "refreshInterval": 3,
   "source": {
      "inline": "ps Axrc|grep -A 1 PID|tail -1|awk -F' ' '{print$5,$6,$7,$8,$9,$10}'"
   },
   "align": "left",
   "background": "#000000",
   "bordered": false,
   "autoResize": true
},
```

**Comment by l0bz1k** [comment]
Thanks, that worked. But ticker line will be better, cause with autoResize option all elements to the right moves to hidden part of touchbar if process name rather long...

**Comment by HigH-HawK** [comment]
True, if you have a set size to fit all your desired widgets onto the touch bar, the `autosize` would break things. The music widget has a `disableMarquee` parameter, which however does not work with `shellScriptTitledButton`

@Toxblh please could you add a `marquee` feature to the `shellScriptTitledButton`? So one can either enable or disable it, if they need it.
